src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/SelfcareApiServiceImpl.scala

```diff
@@ -51,4 +51,24 @@ final case class SelfcareApiServiceImpl(
       }
     }
   }
+
+  override def getInstitutions()(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerSelfcareInstitutionarray: ToEntityMarshaller[Seq[SelfcareInstitution]],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem]
+  ): Route = {
+    val result: Future[Seq[SelfcareInstitution]] =
+      for {
+        userUuid <- getUidFutureUUID(contexts)
+        _ = logger.info(s"Retrieving institutions of user ${userUuid.toString}")
```

Comment:
IMO can be removed because [this](https://github.com/pagopa/interop-be-backend-for-frontend/pull/145/files?diff=split&w=1#diff-8bea4c3bd97f3aed0a0edd0088ec68faf5d088e72424b78f4d9acf0c174ecdccR30) already exists
```suggestion
```